gradle.properties

```diff
@@ -120,7 +120,7 @@ ANDROIDX_COMPOSE_VERSION=1.3.1
 ANDROIDX_CONSTRAINTLAYOUT_VERSION=1.0.1
 ANDROIDX_CORE_VERSION=1.9.0
 ANDROIDX_ESPRESSO_VERSION=3.5.0
-ANDROIDX_LIFECYCLE_VERSION=2.5.1
+ANDROIDX_LIFECYCLE_VERSION=2.6.0-alpha03
```

Comment:
This brings in the new Compose API we need to make flow collection lifecycle aware.